src/render/dom/utils/build-html-styles.ts

```diff
@@ -52,8 +57,8 @@ export function buildHTMLStyles(
     transformKeys.length = 0
 
     // Track whether we encounter any transform or transformOrigin values.
-    let hasTransform = !!isLayoutProjectionEnabled
-    let hasTransformOrigin = !!isLayoutProjectionEnabled
+    let hasTransform = false
+    let hasTransformOrigin = false
```

Comment:
Why don't we base this of the layoutProjectionEnabled anymore?

Reply:
Before the way the later logic was written, we always wanted resolve *some* transform if layout projection was enabled.

Whereas now the logic is slightly different in that if isLayoutProjectionEnabled is true, we still resolve the projection transform, but we *also* want to know if the user has set a transform so we can apply rotate and skew if so.